frontends/precinct-scanner/src/screens/scan_error_screen.tsx

```diff
@@ -37,19 +39,19 @@ export function ScanErrorScreen({
         return 'There was a problem reading this ballot. Please scan again.';
       // Precinct scanner errors
       case 'scanning_failed':
-        return 'Ballot not fully inserted. Remove ballot to continue.';
       case 'both_sides_have_paper':
-        return 'Scanning interrupted. Remove ballot to continue.';
-      case 'paper_in_front_on_startup':
-        return 'Scanner detected an unexpected ballot in the tray. Remove ballot to continue.';
-      case 'paper_in_back_on_startup':
+      case 'paper_in_front_after_reconnect':
+      case 'paper_in_back_after_reconnect':
       case 'paper_in_back_after_accept':
-        return 'Scanner detected an unexpected ballot at the back of the scanner. Remove ballot to continue.';
+        return 'Remove ballot to continue.';
+      case 'paper_status_timed_out':
       case 'scanning_timed_out':
       case 'unexpected_paper_status':
       case 'unexpected_event':
       case 'plustek_error':
-        return 'The scanner experienced an error and needs to be reset. Please turn off and unplug the scanner from the power outlet. Plug it back in and turn it on again.';
+        // These cases require restart, so we don't need to show an error
+        // message, since that's handled below.
+        return undefined;
```

Comment:
@beausmith heads up I made some copy simplifications here. A few reasons:
- The scan service now distinguishes between recoverable and non-recoverable errors better
- The error cases above that had specific reasons the ballot was rejected weren't quite lining up with the reality of what was happening with the scanner, so being a bit more ambiguous makes for a smoother flow

Hopefully in the future we can be more precise, but I think this is the best way forward for now